src/bokeh/server/views/ws.py

```diff
@@ -139,7 +140,7 @@ def open(self) -> None:
             self.close()
             raise ProtocolError("No token received in subprotocol header")
 
-        now = calendar.timegm(dt.datetime.utcnow().utctimetuple())
+        now = calendar.timegm(dt.datetime.now(tz=timezone.utc).utctimetuple())
```

Comment:
According to https://docs.python.org/3/library/datetime.html#datetime.datetime.utctimetuple we can change all calls from `utctimetuple()` to just `timetuple()` now.

Reply:
Added these too